modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CppRestClientCodegen.java

```diff
@@ -153,13 +138,45 @@ public CppRestClientCodegen() {
         typeMapping.put("UUID", "utility::string_t");
 
         super.importMapping = new HashMap<String, String>();
+        updateImportMapping();
+
+        addSupportingFiles();
```

Comment:
How about removing the two calls here, and have them only in `processOpts` ?

Reply:
Can I be sure that processOpts() will always be called?

Reply:
I also don't know that codebase well enough, but afaict it will always be called.
To put it differently, if it _isn't_ called there'll be a lot of issue with the generated code anyway.
I just think it's better to avoid duplicating calls to functions like these ones, again for maintainability purposes. Their execution directly depends on an option being passed, so it makes more sense to call them only when that option is known.

Reply:
OK, then I'll simplify it.